typescript/src/schema/resourceDatabase.ts

```diff
@@ -21,8 +21,8 @@ export interface ResourceDescriptor
 }
 
 export class ResourceDatabase {
-  private patterns: { [pattern: string]: ResourceDescriptor };
-  private types: { [type: string]: ResourceDescriptor };
+  patterns: { [pattern: string]: ResourceDescriptor };
```

Comment:
why not private anymore?

Reply:
It's being used here in `proto.ts`:
`const allPatterns = resourceDatabase.patterns;`